src/renderer/components/dock/pod-logs.store.ts

```diff
@@ -1,33 +1,38 @@
-import { autorun, observable } from "mobx";
-import { Pod, IPodContainer, podsApi } from "../../api/endpoints";
+import { autorun, computed, observable, reaction } from "mobx";
+import { Pod, IPodContainer, podsApi, IPodLogsQuery } from "../../api/endpoints";
 import { autobind, interval } from "../../utils";
 import { DockTabStore } from "./dock-tab.store";
 import { dockStore, IDockTab, TabKind } from "./dock.store";
 import { t } from "@lingui/macro";
 import { _i18n } from "../../i18n";
+import { Notifications } from "../notifications";
+import { isDevelopment } from "../../../common/vars";
 
 export interface IPodLogsData {
   pod: Pod;
   selectedContainer: IPodContainer
   containers: IPodContainer[]
   initContainers: IPodContainer[]
   showTimestamps: boolean
-  tailLines: number
   previous: boolean
 }
 
 type TabId = string;
+type PodLogs = string;
 
-interface PodLogs {
-  oldLogs?: string
-  newLogs?: string
-}
+// Number for log lines to load
+export const logRange = isDevelopment ? 100 : 1000;
```

Comment:
Why this value different for `development`?

Reply:
Just for testing reasons, it's easier to play with logs when you have smaller amount of it.